src/flask/debughelpers.py

```diff
@@ -143,20 +148,21 @@ def explain_template_loading_attempts(app, template, attempts):
         for line in _dump_loader_info(loader):
             info.append("       %s" % line)
 
-        if triple is None:
-            detail = "no match"
-        else:
+        detail = "no match"
+        if triple is not None:
             detail = "found (%r)" % (triple[1] or "<string>")
             total_found += 1
+
         info.append("       -> %s" % detail)
 
     seems_fishy = False
```

Comment:
i think you dont need this.

Reply:
Yes, I think so. I totally forgot about it. I'll commit again.